astroid/brain/brain_attrs.py

```diff
@@ -6,9 +6,9 @@
 Without this hook pylint reports unsupported-assignment-operation
 for attrs classes
 """
-
-import astroid
 from astroid import MANAGER
+from astroid.node_classes import AnnAssign, Assign, Call, Unknown
+from astroid.scoped_nodes import ClassDef
```

Comment:
I still think it would make sense to import those from `astroid.nodes` instead. Even more so if we want to reorganize `node_classes` in the future. Might make sense to split those 5000 lines into multiple files inside a sub-package (called nodes?).

Reply:
>  Might make sense to split those 5000 lines into multiple files inside a sub-package (called nodes?).

Yes, I'm working on it ! 

But right now there are cyclic import that prevent to import directly from nodes without hacks (and will for a long time seeing how it goes right now). I want to do this in multiple small steps :) So yes the end goal would be to import from ``astroid.nodes``

Reply:
Original idea here : https://github.com/PyCQA/astroid/pull/587#issuecomment-403326307

Reply:
Sounds good. I would just suggest to do it in smaller pieces. That will make resolving conflicts easier.